src/main/java/nextstep/subway/line/domain/Distance.java

```diff
@@ -0,0 +1,46 @@
+package nextstep.subway.line.domain;
+
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Distance {
+    public static final int MIN_DISTANCE = 0;
+    private int distance;
+
+    public Distance() {
+    }
+
+    public Distance(int distance) {
+        validateDistance(distance);
+        this.distance = distance;
+    }
+
+    private void validateDistance(int distance) {
+        if (distance <= MIN_DISTANCE) {
+            throw new IllegalArgumentException("지하철 구간 사이의 거리는 " + MIN_DISTANCE + "보다 커야 합니다.");
+        }
+    }
+
+    public boolean divisible(Distance divideDistance) {
+        if (distance <= divideDistance.getDistance()) {
+            throw new IllegalArgumentException("역과 역 사이의 거리보다 좁은 거리를 입력해주세요");
+        }
+        return true;
+    }
+
+    public Distance addDistance(Distance newDistance) {
+        return new Distance(this.distance + newDistance.distance);
+    }
```

Comment:
아래 이유들을 생각해보면 add 보다는 plus가 더욱 어울리는 메서드 명일 수도 있을 것 같아요 ! 🤔 
어떻게 생각하시나요 ?
1. 거리를 빼는 메서드 명을 minus를 사용
2. add는 동사고 plus는 명사인데, 보통 동사는 내부의 상태를 변경하는 메서드 명으로 주로 사용됩니다.

```suggestion
    public Distance plus(Distance newDistance) {
        return new Distance(this.distance + newDistance.distance);
    }
```

Reply:
넵 동의합니다..! 메서드명 ``plus``로 수정하였습니다.